docs/reference-architectures/ai/realtime-scoring-python.md

```diff
@@ -2,7 +2,7 @@
 title: Real-time scoring of Python models
 titleSuffix: Azure Reference Architectures
 description: This reference architecture shows how to deploy Python models as web services on Azure to make real-time predictions.
-author: njray
+author: masalvar
```

Comment:
author field should be a GitHub alias. (msalvaris?)

Reply:
Ah, ok. Yes msalvaris is the github one